src/main/webapp/app/tasks/local-task.service.spec.ts

```diff
@@ -6,7 +6,8 @@ import { Task } from './task';
 describe('LocalTaskService', () => {
   const id = 'de4f576e-d1b5-488a-8c77-63d4c8726909';
   const name = 'Doing the do!';
-  const mockTask = `{"id":"${id}","name":"${name}"}`;
+  const completed = false;
+  const mockTask = `{"id":"${id}","name":"${name}", "completed":"${completed}"}`;
```

Comment:
the value of  mockTask completed is assigned as string and will be interpreted as such, making '**should return tasks from local storage**' fail with _Expected 'false' to equal false._

Reply:
That's right! Somehow I didn't notice... I've tried to fix the issue with the tests, but I can't seem to find the problem. Do you have a suggestion?